packages/eslint-plugin/docs/rules/boolean-literal-compare.md

```diff
@@ -0,0 +1,34 @@
+# Flags unnecessary equality comparisons against booleans (boolean-literal-compare)
+
+Comparing boolean values to boolean literals is unnecessary, as those expressions will result in booleans too.
+Just use the boolean values directly or negate them.
```

Comment:
Probably better as something like

---

Comparing boolean values to boolean literals is unnecessary, those comparisons result in the same booleans. Using the boolean values directly, or via a unary negation (`!value`) is more concise and clearer.